HireUp-Turing/HireUp_frontend
# Feature/employer messages

#### Type of Change Made 
- [ ] formatting 
#### What's this PR do?
* Allows users to send messages to applicants
* Prevents applicants from messaging themselves 
* Handles several sad-path message actions
* Says 'You don't have any messages yet' if inbox is empty
* Only allows users to view their own inbox
#### Where should the reviewer start?
* MessageForm.tsx
#### How should this be manually tested?
Send some messages, go check the users inbox. (You may have to use the component inspector to fake a log in)
#### What are the relevant tickets?
close #54
